integration/metrics_test.go

```diff
@@ -106,15 +106,15 @@ func TestOverlayFallbackMetric(t *testing.T) {
 			name:  "image with all layers having ztocs and no fs.Mount error results in 0 overlay fallback",
 			image: rabbitmqImage,
 			indexDigestFn: func(sh *shell.Shell, image imageInfo) string {
-				return buildSparseIndex(sh, image, 0, defaultSpanSize)
+				return buildIndex(sh, image, withMinLayerSize(0), withSpanSize(defaultSpanSize))
```

Comment:
`withMinLayerSize(0), withSpanSize(defaultSpanSize)` can be removed.

Reply:
Agreed on `withSpanSize`. I opted to keep `withMinLayerSize` because that is important to this test. I think it's good to be explicit when it's relevant, even if it's a no-op. Thoughts?

Reply:
SGTM either explicit or implicit.

Meanwhile, maybe we should take both `defaultSpanSize` and `defaultMinLayerSize` as the default for `buildIndex` which is consistent with cli. Then if we anything other than default we should call out (e.g., `withMinLayerSize(0)`).

Currently `buildIndex` uses `defaultSpanSize` as default for span size, but `0` for `minLayerSize` which is confusing, given that we have the `defaultMinLayerSize` and the func name `buildIndex` is generic (i.e., not `buildSparseIndex` or `buildFullIndex`)

Reply:
I think we can switch that in a follow up. It will also give us a chance to look at the tests and figure out which should have a `spanSize` set to 0 and which shouldn't.

Reply:
SGTM. We can open an issue for that or just keep #349 opened.

Reply:
I updated #349 to include updating those.